includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -920,13 +927,15 @@ public static function orders( $data = null ) {
 
 				// Add the adjustment.
 				$adjustment_args = array(
-					'object_id'   => $order_id,
-					'object_type' => 'order',
-					'type'        => 'fee',
-					'description' => $fee['label'],
-					'subtotal'    => floatval( $fee['amount'] ),
-					'tax'         => $tax,
-					'total'       => floatval( $fee['amount'] ) + $tax,
+					'object_id'     => $order_id,
+					'object_type'   => 'order',
+					'type'          => 'fee',
+					'description'   => $fee['label'],
+					'subtotal'      => floatval( $fee['amount'] ),
+					'tax'           => $tax,
+					'total'         => floatval( $fee['amount'] ) + $tax,
+					'date_created'  => $date_created_gmt,
```

Comment:
Should this be `$data->post_modified_gmt`? If not, does it deserve an inline comment because it's different?